Movie/Movie/Services/Network/Models/RecommendationMovieResponse.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  RecommendationMovie.swift
+//  Movie
+//
+//  Created by Алена Панченко on 28.10.2022.
+//
+
+import Foundation
+
+/// Модель массива рекоммендации
+struct RecommendationMovieResponse: Decodable {
+    enum CodingKeys: String, CodingKey {
```

Comment:
```suggestion
    enum CodingKeys: String, CodingKey {
```
 это ниже полей в моделе так